.github/workflows/release.yml

```diff
@@ -9,21 +9,17 @@ jobs:
 
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [14.x]
-
     steps:
     - uses: actions/checkout@v3
     - name: Publish to npm
       uses: actions/setup-node@v3
       with:
-        node-version: ${{ matrix.node-version }}
+        node-version: lts/*
```

Comment:
this will currently resolve to v18. Not sure if you wanna lock it down or not? Either way, a matrix is not needed here